pkg/metadata/gohai/gohai.go

```diff
@@ -47,7 +55,8 @@ func getGohaiInfo() *gohai {
 		log.Errorf("Failed to retrieve memory metadata: %s", err)
 	}
 
-	if !config.IsContainerized() {
+	detectDocker0()
```

Comment:
nit, you could combine this with returning the var so that you could write `if docker0Detected()`

Reply:
Good point, done